examples/kubernetes/templates/v1/cilium-ds.yaml.sed

```diff
@@ -32,6 +32,22 @@ spec:
         prometheus.io/port: "9090"
     spec:
       serviceAccountName: cilium
+      initContainers:
+      - name: clean-cilium-state
+        image: busybox
+        command: ['sh', '-c', 'if [ ${CLEAN_CILIUM_STATE} ]; then rm -rf /var/run/cilium/state; rm -rf /sys/fs/bpf/tc/globals/cilium_*; fi']
```

Comment:
If the option is set to `false` this doesn’t seem to do the right thing.
This should be:
```
... if [ “${CLEAN_CILIUM_STATE}” = “true” ] ...
```

Reply:
yes.. fixing it

Reply:
And if you wanted to check whether the variable was set, this test should have been different anyway:
```
if [ -n "${CLEAN_CILIUM_STATE}" ] ...
```